### ISSUE TITLE
Page view window issues

### ISSUE THREAD

**Issue by reidmporter** [OpenDataServices/org-ids#143]
Chrome Version 57.0.2987.133 (64-bit)

On "small" (13" MBP) screens, SEE LISTS and About bar not visible unless full screen or zoomed out.

![image](https://user-images.githubusercontent.com/15111135/29122586-d804a7bc-7ce0-11e7-90f0-dc3183d13c80.png)
![image](https://user-images.githubusercontent.com/15111135/29122595-e3d60a04-7ce0-11e7-9cf9-e47d6cee78f9.png)

**Comment by andylolz** [comment]
Surprised about bar isn’t visible… You’re definitely viewing at 100% zoom, @reidmporter?

**Comment by reidmporter** [comment]
Yup. Tried to make a video showing how they disappear and reappear but got distracted. Expanding window edge to edge solves button but not bar.

**Comment by timgdavies** [comment]
Flagging to @ccmdesign as a snagging issue
